README.md

```diff
@@ -1927,6 +1927,39 @@ no parameters.
   `lib/hello_world/hello_world.rb`.
 <sup>[[link](#snake-case-dirs)]</sup>
 
+* <a name="do-not-abbreviate"></a>
```

Comment:
I am definitely on-board in principle here, but I am also on board with 120 line lengths. If we want both short lines and long var names, can we add a section on the appropriate line-breaking style? And maybe refer to that in here.